core/monitorserver/server.go

```diff
@@ -86,19 +87,19 @@ func (s *Server) getResponseByRevision(epoch int64) (*pb.State, error) {
 		return nil, grpc.Errorf(codes.NotFound, "Could not find monitoring response for epoch %d", epoch)
 	}
 
-	resp := &pb.State{
-		Smr:                r.Smr,
-		SeenTimestampNanos: r.Seen.UnixNano(),
-	}
-
+	// Convert errors into rpc.Status
+	errs := make([]*statuspb.Status, 0, len(r.Errors))
```

Comment:
(optional, since this is a different package, and exporting `errList` makes you extra work)
This block which maps `[]error -> []*statuspb.Status` would be nice as a func on `errList`